### PR TITLE
Issue 150: add regional publication facility / centre identifier clarification

### LINKED ISSUE #150: validate metadata centre-id mismatch
As part of WIS2 Global Services Testing, we have defined a [GDC functional test](https://wmo-im.github.io/wis2-global-services-testing/global-services-testing/wis2-global-services-testing-DRAFT.html#_metadata_ingest_centre_id_mismatch) to reject/discard a WCMP2 record if the centre-id component of its `id` property does not match the centre-id of the topic.

We need to validate whether this is desired behaviour.  For example, a data publisher can have its WCMP2 record published by a broker from another centre as a "regional publication" function/facility as a valid workflow.

cc @shirley-xuelei

### PR DESCRIPTION
**Pull request by tomkralidis**
- Fixes #150 
- Fixes minor AsciiDoc typo

### CODE REVIEW

File: guide/sections/part2/wis2node.adoc [hunk 1]
```diff
@@ -8,6 +8,8 @@ Registration and decommissioning of WIS2 Nodes must be approved by the Permanent
 
 A WIS2 Node can be registered to exchange data concerning a WMO project or campaign, for a limited time. The National Focal Point can register such a project-related WIS2 Node in coordination with the Secretariat.
 
+A WIS2 Node can act as a regional publication facility on behalf of other centres.  Data or metadata publication by a regional publication facility will use the centre identifier of the data publisher.
```
Comment (golfvert) [inline guide/sections/part2/wis2node.adoc:11]: "regional publication facility" is not a concept that exists in WMO. The correct name is a DCPC (the DC part).
And "data publisher" might be confusing. I guess we are speaking about the "data producer" / "data originator". No ?

What about:

A WIS2 Node can act as a publication facility on behalf of other centres. This is a Data Collection role, as defined in the Manual on WIS. Data or metadata publication by a Data Collection will use the centre identifiers of the data producers.

Comment (tomkralidis) [reply guide/sections/part2/wis2node.adoc]: OK.  Updated to:

A WIS2 Node can act as a publication facility on behalf of other centres. This is a Data Collection role, as defined in the Manual on WIS. Data or metadata publication by a Data Collection role will use the centre identifiers of the data producers.

File: guide/sections/part2/wis2node.adoc [hunk 2]
```diff
@@ -8,7 +8,7 @@ Registration and decommissioning of WIS2 Nodes must be approved by the Permanent
 
 A WIS2 Node can be registered to exchange data concerning a WMO project or campaign, for a limited time. The National Focal Point can register such a project-related WIS2 Node in coordination with the Secretariat.
 
-A WIS2 Node can act as a regional publication facility on behalf of other centres.  Data or metadata publication by a regional publication facility will use the centre identifier of the data publisher.
+A WIS2 Node can act as a publication facility on behalf of other centres. This is a Data Collection role, as defined in the Manual on WIS. Data or metadata publication by a Data Collection role will use the centre identifiers of the data producers.
```
Comment (golfvert) [inline guide/sections/part2/wis2node.adoc:11]: On the second "Data Collection role" above, I wouldn't have role (as it was the case in my proposal). OK for you ?
A DCPC is a thing. We don't say GISC role, just GISC...

Comment (tomkralidis) [reply guide/sections/part2/wis2node.adoc:11]: Ok. Then we should qualify DCPC proper.

Comment (tomkralidis) [reply guide/sections/part2/wis2node.adoc:11]: How about:

A WIS2 Node can act as a publication facility on behalf of other centres. This is a Data Collection or Production Centre (DCPC) role, as defined in the Manual on WIS. Data or metadata publication by a DCPC will use the centre identifiers of the data producers.

Comment (golfvert) [reply guide/sections/part2/wis2node.adoc:11]: OK

Comment (tomkralidis) [reply guide/sections/part2/wis2node.adoc]: PR updated.

**Review by golfvert — APPROVED** [review]

**Pull request merged**